doc/main.dox

```diff
@@ -119,6 +119,43 @@ Thus the following example specifies all sensors for four combinations of visit
 plus all sensors for one raft of two other visits for calibrated lsstSim data:
 - `--id visit=54123^55523 raft=1,1^2,1 --id visit=623459^293423 raft=0,0`
 
+\subsection pipeBase_argumentParser_rerun The --rerun Option
+
+The `--rerun` option is a convenient shortcut for specifying the output directory of a command line task and
+serves the same purpose as the `--output` argument. However, the value supplied to `--rerun` is relative to
+the "rerun" directory of the root input data repository (i.e. follow the "_parent" chain all the way back,
+then add "rerun" to that). `--rerun` saves the user from typing in the input repository twice. For example:
```

Comment:
This is pretty good, but I think it could be clearer. How about starting out with something like: "The --rerun option is an alternate way to specify the output (and optionally the input) repositories for a command line task. Unlike `--output`, the value supplied to `--rerun` is relative to..." (where the rest is as you wrote it).

I think it would be clearer and is less likely to trouble the user with the nagging question "what if I specify --output and --rerun", since it's pretty clear that will be an error, or at least a bad idea.